My project use react-doc-viewer error Can't resolve 'csv-parse/browser/esm'

Module not found: Can't resolve 'csv-parse/browser/esm' in 'D:\Teu\development-project\Client\node_modules\@cyntler\react-doc-viewer\dist\esm\plugins\csv'

--- a/package.json
+++ b/package.json
@@ -18,11 +18,6 @@
   "main": "./dist/cjs/index.js",
   "module": "./dist/esm/index.js",
   "types": "./dist/esm/index.d.ts",
-  "jest": {
-    "moduleNameMapper": {
-      "^csv-parse/browser/esm": "<rootDir>/node_modules/csv-parse/dist/cjs/index.cjs"
-    }
-  },
   "engines": {
     "node": ">=12.0.0"
   },
@@ -38,8 +33,9 @@
   },
   "dependencies": {
     "@types/mustache": "^4.2.1",
-    "csv-parse": "^5.3.2",
+    "@types/papaparse": "^5.3.5",
     "mustache": "^4.2.0",
+    "papaparse": "^5.3.2",
     "react-pdf": "5.7.2",
     "styled-components": "^5.3.6",
     "wl-msg-reader": "^0.2.0"

--- a/src/plugins/csv/index.tsx
+++ b/src/plugins/csv/index.tsx
@@ -1,6 +1,6 @@
 import React, { useEffect, useState } from "react";
 import styled from "styled-components";
-import { parse } from "csv-parse/browser/esm";
+import { parse } from "papaparse";
 import { DocRenderer } from "../..";
 import { textFileLoader } from "../../utils/fileLoaders";
 
@@ -11,17 +11,13 @@ const CSVRenderer: DocRenderer = ({
 
   useEffect(() => {
     if (currentDocument?.fileData) {
-      parse(
-        currentDocument.fileData as string,
-        {
-          delimiter: config?.csvDelimiter ?? ",",
-        },
-        (err, records) => {
-          if (!err && records) {
-            setRows(records);
-          }
-        }
-      );
+      const parseResult = parse(currentDocument.fileData as string, {
+        delimiter: config?.csvDelimiter ?? ",",
+      });
+
+      if (!parseResult.errors?.length && parseResult.data) {
+        setRows(parseResult.data as string[][]);
+      }
     }
   }, [currentDocument, config?.csvDelimiter]);
 

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -11,7 +11,7 @@
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "moduleResolution": "node",
+    "moduleResolution": "Node",
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "react",

